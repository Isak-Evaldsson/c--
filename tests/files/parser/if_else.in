int test()
{
    if (1) {
        print(1);
    } else {
        print(2);
    }

    if (1)
        print(1);
    else {
        print(2);
    }
  
    if (1) {
        print(1);
    } else
        print(2);

    if (1)
        print(1);
    else
        print(2);
}