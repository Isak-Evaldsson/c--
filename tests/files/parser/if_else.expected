Prototype: test
    If:
        LiteralExpr: 1
    Do:
        FunCall: print
            LiteralExpr: 1
    Else:
        FunCall: print
            LiteralExpr: 2
    If:
        LiteralExpr: 1
    Do:
        FunCall: print
            LiteralExpr: 1
    Else:
        FunCall: print
            LiteralExpr: 2
    If:
        LiteralExpr: 1
    Do:
        FunCall: print
            LiteralExpr: 1
    Else:
        FunCall: print
            LiteralExpr: 2
    If:
        LiteralExpr: 1
    Do:
        FunCall: print
            LiteralExpr: 1
    Else:
        FunCall: print
            LiteralExpr: 2
